wire-ios-data-model/Source/Model/UserClient/E2eIdentityCertificate.swift

```diff
@@ -23,6 +23,14 @@ public enum E2EIdentityCertificateStatus: CaseIterable {
     case notActivated, revoked, expired, invalid, valid
 }
 
+public enum E2eIdentityCertificateConstants {
```

Comment:
Since the constants are only used in this file, shouldn't they be `private` instead?
Also, couldn't we move this enum below the main content of this file (`class E2eIdentityCertificate`)?

Reply:
Enum 'E2eIdentityCertificateConstants' is internal and cannot be referenced from a default argument value.
I like to keep the constants which are dependencies on top of the file as it s easier to reference.